google/ggrc-core
# GGRC-6573 Disable notifications for external models

# Issue description

Disable notifications for external models. Disabled notification creation for:
- Comment with notifications
- Review notification

# Steps to test the changes
Comment notification test:
1. Check all notifications count in DB
2. Create control using external app user
3. Create comment with notification checked using internal user
4. After creation of comment number of notifications in DB should be the same

Review notification test:
1. Check all notifications count in DB
2. Create control using external app user
3. Mark control as reviewed using internal user
4. Change some attributes in control using external app user
5. After modification of control number of notifications in DB should be the same

# Solution description

I prevented notification creation for models with Synchronizable mixin in comment creation flow and review notification flow.

# Sanity checklist




# PR Review checklist
